docs/src/intro-csharp.md

```diff
@@ -122,6 +122,25 @@ dotnet test -- NUnit.NumberOfTestWorkers=5
 pwsh bin\Debug\netX\playwright.ps1 codegen
 ```
 
+## Bundle drivers for different platforms
```

Comment:
I don't think it needs to me this high in the list, probably closer to the end, say that "By default, Playwright will only bundle..."

Reply:
it can't be lower.